packages/ddex/processor/src/parseDelivery.ts

```diff
@@ -282,35 +283,42 @@ function parseReleaseXml(source: string, $: cheerio.CheerioAPI) {
         return
       }
 
+      // add deal
+      function addDeal(deal: AudiusSupportedDeal) {
+        releaseDeals[ref] ||= []
+        releaseDeals[ref].push(deal)
+      }
+
       const common: DealFields = {
-        isDownloadable: usageTypes.includes('PermanentDownload'),
+        forStream: usageTypes.includes('OnDemandStream'),
```

Comment:
we also accept 'Stream' per the audius docs https://docs.audius.org/distributors/specification/deal-types/